data/src/main/java/com/teamagam/gimelgimel/data/message/rest/RestAPI.java

```diff
@@ -189,10 +190,10 @@ private Gson createMessagingGson() {
         MessageJsonAdapter messageJsonAdapter = new MessageJsonAdapter();
         return new GsonBuilder()
                 .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+                .registerTypeAdapterFactory(new GeometryAdapterFactory())
                 .registerTypeAdapter(MessageData.class, messageJsonAdapter)
-                .registerTypeAdapter(List.class,
-                        new MessageListJsonAdapter(messageJsonAdapter))
-                .registerTypeAdapter(Coordinate.class, new Coordinate.CoordinateSerializer())
+                .registerTypeAdapter(List.class, new MessageListJsonAdapter(messageJsonAdapter))
+                //.registerTypeAdapter(Coordinate.class, new Coordinate.CoordinateSerializer())
```

Comment:
delete comment.
It seems like the class Coordinate and CoordinateSerializer aren't used anymore, they can also be deleted (with its tests)